mobile/src/screens/survey/libs/constants.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  PREFERENCES_OTHER_CATEGORY,
+  type SurveyInputDto,
+} from '#packages/survey/survey';
+
+const TEXTAREA_MAX_LENGTH = 1000;
+const TEXTAREA_ROWS_COUNT = 3;
+
+const INVALID_ARRAY_INDEX = -1;
```

Comment:
We already have current contant for frontend side. Let's move to shared. Please contact with developer of frontend side

Reply:
I think we should keep logic implementation separately, it may differ for mobile and frontend

Reply:
this is mobile specific implementation, let's leave it

Reply:
except INVALID_ARRAY_INDEX, we no longer need it, so it should be removed

Reply:
@v0ldemar01 you were right, I missunderstood your comment 😅